style.css

```diff
@@ -0,0 +1,103 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  outline: none;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  background: burlywood;
+}
+
+.wrapper {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.todo-list {
+  margin: 10% 0 0 0;
+  display: flex;
+  justify-content: center;
+  width: 50%;
+  flex-direction: column;
+}
+
+.todo-list__add {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 10px solid yellow;
+  border-radius: 10px;
+  padding: 3%;
+  background: skyblue;
+}
+
+.todo-list__input {
+  width: 100%;
+  margin: 0 3% 0 0;
+}
+
+.todo-list__input input {
+  width: 100%;
+  height: 35px;
```

Comment:
Well, we've already talked about units. Use at least either only absolute or only relative.
And if somewhere you still had to use a different type of units, then write a comment why